src/specialOffer/SpecialOffer.style.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+
+import { responsiveBreakpoints, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+
+export const StyledSpecialOffer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  padding: ${space.m};
+`
+
+export const StyledSpecialOfferContent = styled.div`
+  flex: 1;
+  align-items: flex-start;
+  max-width: 50%;
+
+  @media (${responsiveBreakpoints.isMediaSmall}) {
+    max-width: 100%;
+    min-width: 100%;
+  }
+`
+
+export const StyledSpecialOfferFigure = styled.figure`
+  width: 70%;
+  padding-top: ${space.xl};
+  padding-bottom: ${space.m};
+  margin: 0;
+
+  ${normalizeHorizontally}
```

Comment:
Why do you need normalization?

Reply:
It's for the `figure` to have the same margins as the `SubHeader` or the `Paragraph` below.

With normalization: 

![image](https://user-images.githubusercontent.com/748586/109018092-1a2aba80-76b8-11eb-85fe-a162a443380e.png)


Without:

![image](https://user-images.githubusercontent.com/748586/109018123-21ea5f00-76b8-11eb-9f75-f072536f265a.png)